xbmc/PlayListPlayer.cpp

```diff
@@ -989,43 +999,42 @@ void PLAYLIST::CPlayListPlayer::OnApplicationMessage(KODI::MESSAGING::ThreadMess
     g_application.WakeUpScreenSaverAndDPMS();
 
     // stop playing file
-    if (g_application.GetAppPlayer().IsPlaying())
+    if (appPlayer->IsPlaying())
```

Comment:
Maybe stop playing can also be encapsulated?

Reply:
coming later in the series.